cloudformation/memsub-promotions-lambdas-cf.yaml

```diff
@@ -248,7 +285,36 @@ Resources:
       FunctionName: MembershipSub-PromoCode-View-Dynamo-to-Salesforce-PROD
       Handler: MembershipSub-PromoCode-View-Dynamo-to-Salesforce.handler
       MemorySize: 128
-      Role: !GetAtt [MembershipSubPromotionsDataRole, Arn]
+      Role: !GetAtt [MembershipSubPromoCodeViewETLRole, Arn]
       Runtime: nodejs6.10
       Timeout: 10
-      KmsKeyArn: !GetAtt SalesforceLambdaKMSKey.Arn
\ No newline at end of file
+      KmsKeyArn: !GetAtt SalesforceLambdaKMSKey.Arn
+  ScheduledBackupLambdaPROD:
+    Type: AWS::Lambda::Function
+    Properties:
+      Code:
+        S3Bucket: gu-promotions-tool-dist
+        S3Key: membership/PROD/MembershipSub-Promotions-PromoCode-View/MembershipSub-Promotions-PromoCode-View.zip
+      Description: A Lambda function to schedule the backup of the Campaigns and Promotions table
+      FunctionName: MembershipSub-Promotions-Scheduled-Dynamo-Backup-PROD
+      Handler: MembershipSub-Promotions-Scheduled-Dynamo-Backup.handler
+      MemorySize: 128
+      Role: !GetAtt [MembershipSubPromotionsDataBackupLambdaRole, Arn]
+      Runtime: nodejs6.10
+      Timeout: 10
+      KmsKeyArn: !GetAtt SalesforceLambdaKMSKey.Arn
+  BackupSchedule:
+    Type: AWS::Events::Rule
+    Properties:
+      ScheduleExpression: cron(0 12 * * ? *)
```

Comment:
minor: I think rate(x) is easier to understand than cron()

https://docs.aws.amazon.com/lambda/latest/dg/tutorial-scheduled-events-schedule-expressions.html

Reply:
I'm going to keep it as cron, because I want to specify a time of day.